docs/sources/flow/getting-started/migrating-from-static.md

```diff
@@ -0,0 +1,322 @@
+
+> In this task, we will use the [run][] CLI command to run Grafana Agent in Flow
+> Mode using a Grafana Agent Static Mode configuration.
+
+[Start the Agent][] in flow mode and include the command line flag
+`--config.format=static`. Your configuration file must be a valid [Static]
+configuration file rather than a Flow Mode configuration file.
+
+### Debugging
+
+1. You can follow the convert CLI command [debugging][] instructions to generate
+   a diagnostic report.
+
+1. Refer to the Grafana Agent [Flow Debugging][] for more information about
+   running Grafana Agent in Flow Mode.
+
+1. If your [Static] configuration cannot be converted and loaded directly into
+   Grafana Agent, diagnostic information is sent to `stderr`. You can bypass any
+   non-critical issues and start the Agent by including the
+   `--config.bypass-conversion-errors` flag in addition to
+   `--config.format=static`.
+
+   {{% admonition type="caution" %}}If you bypass the errors, the behavior of the converted configuration may not match the original Grafana Agent Static Mode configuration. Do not use this flag in a production environment.{{%/admonition %}}
+
+## Example
+
+This example demonstrates converting a [Static] configuration file to a Grafana
+Agent Flow Mode configuration file.
+
+The following [Static] configuration file provides the input for the conversion:
+
+```yaml
+server:
+  log_level: info
+
+metrics:
+  global:
+    scrape_interval: 15s
+    remote_write:
+      - url: https://prometheus-us-central1.grafana.net/api/prom/push
+        basic_auth:
+          username: USERNAME
+          password: PASSWORD
+  configs:
+    - name: test
+      host_filter: false
+      scrape_configs:
+        - job_name: local-agent
+          static_configs:
+            - targets: ['127.0.0.1:12345']
+              labels:
+                cluster: 'localhost'
+    
+logs:
+  global:
+    file_watch_config:
+      min_poll_frequency: 1s
+      max_poll_frequency: 5s
+  positions_directory: /var/lib/agent/data-agent
+  configs:
+    - name: varlogs
+      scrape_configs:
+        - job_name: varlogs
+          static_configs:
+            - targets:
+              - localhost
+              labels:
+                job: varlogs
+                host: mylocalhost
+                __path__: /var/log/*.log
+          pipeline_stages:
+            - match:
+                selector: '{filename="/var/log/*.log"}'
+                stages:
+                - drop:
+                    expression: '^[^0-9]{4}'
+                - regex:
+                    expression: '^(?P<timestamp>\d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2}) \[(?P<level>[[:alpha:]]+)\] (?:\d+)\#(?:\d+): \*(?:\d+) (?P<message>.+)$'
+                - pack:
+                    labels:
+                      - level
+      clients:
+        - url: https://USER_ID:API_KEY@logs-prod3.grafana.net/loki/api/v1/push
+```
+
+The convert command takes the YAML file as input and outputs a River file.
```

Comment:
So far we don't explain River files... it's always referred to as a Flow mode configuration file. Do we want to keep using Flow mode configuration? Or define/link and use River?  

In this case I'd probably go with Flow mode configuration file (that is written in River... similar to how a Static mode file is written in YAML). 

Would it be useful to mention/link to the River description?

Reply:
I think a link here for River makes sense. I can add it to each of the 3 migration guides.